src/twisted/conch/ssh/keys.py

```diff
@@ -1029,16 +1058,22 @@ def sshType(self):
         @return: The key type format.
         @rtype: L{bytes}
         """
+        if self._sk:
```

Comment:
We might want to add a link to the docs https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.u2f and mention that only these 2 key types are supported

Reply:
Added a check with an exception if we are trying to load a certificate based key. Added also the related test and the mentioned comment.

Reply:
codeco.io still complains about a missing  branch coverate at line 1071

https://app.codecov.io/gh/twisted/twisted/pull/12300?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=twisted#f5b617ef489613e9340d60698a081044-R1068

Reply:
Do you have any suggestion about the coverage for this implicit `else` ?

Reply:
@adiroiban 

To solve the issue, I have three proposals: 

1.  we can mock up the `fromString` function returning an SK key that is not supported 
2. we can manually set the SK bit to True of a DSS key for instance to reach the implicit else 
```
        # Test for the implicit else branch
        # create a DSS key with sk flag to True
        skKey = keys.Key(self.dsaObj)
        skKey._sk = True
        self.assertEqual(
            skKey.supportedSignatureAlgorithms(),
            [b"ssh-dss"],
        )
```
3. we can add a comment to the branch to be skipped by the coverage algorithm